app-dicts/myspell-en/metadata.xml

```diff
@@ -10,4 +10,7 @@
 		<name>Proxy Maintainers</name>
 	</maintainer>
 	<longdescription>English dictionaries for OpenOffice/LibreOffice 4.0+ in an individual extension.</longdescription>
+	<use>
+    	<flag name="l10n_en-CA">English (Canada)</flag>
```

Comment:
Those flags aren't supposed to be defined locally. Add it to global definitions.